apps/docs/content/guides/getting-started/quickstarts/hono.mdx

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
I'm wondering if we should have something more than just calling the bootstrap command. The bootstrap command is awesome, but following this guide doesn't really teach anything about how to use Supabase with Hono.